diff --git a/node_modules/expo/src/Expo.fx.tsx b/node_modules/expo/src/Expo.fx.tsx
index 4d8e338..936b73f 100644
--- a/node_modules/expo/src/Expo.fx.tsx
+++ b/node_modules/expo/src/Expo.fx.tsx
@@ -1,6 +1,6 @@
 // load expo-asset immediately to set a custom `source` transformer in React Native
-import './winter';
-import './async-require';
+import './winter/index.ts';
+import './async-require/index.ts';
 import 'expo-asset';
 import 'expo/virtual/rsc';
 
@@ -17,7 +17,7 @@ if (
   // Disable for SSR
   typeof globalThis.expo !== 'undefined'
 ) {
-  require('./async-require/messageSocket');
+  require('./async-require/messageSocket.native.ts');
 }
 
 if (isRunningInExpoGo()) {
